resources/js/features/facility-edit/FacilityForm.tsx

```diff
@@ -9,47 +9,50 @@ function getUrlSuggestion(name: string) {
   const url = trimInput(name).toLowerCase().replaceAll(" ", "-");
   // Remove diacritics, especially for polish characters.
   // https://stackoverflow.com/a/37511463/1832228
-  return url.normalize("NFD").replace(/\p{Diacritic}/gu, "").replace('ł', 'l');
+  return url
```

Comment:
Hm, ta linia to jest dziwne że ci się zawinęła. Mamy w configu prettiera długość linii chyba ze 120 i to na pewno wchodziło. Nie żebym lubił prettiera, prettier to gówno, tylko się dziwię co tu się stało.

Reply:
Nie wiem dlaczego, ale zarówno npm run format jak i vscode mi to zawija.